src/components/HealthCardVerify/HealthCardVerify.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import axios from 'axios';
+import https from 'https';
+import jose from 'node-jose';
+import {
+  Typography
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+  bold: {
+    fontWeight: 'bold'
+  }
+});
+
+const HealthCardVerify = ({ vc }) => {
```

Comment:
```suggestion
const HealthCardVerify = ({ jws, iss }) => {
```
Having multiple props Instead of a single prop that's a complex object makes it more obvious what the inputs to the component are.

Reply:
Updated to split objects